src/index.ts

```diff
@@ -148,9 +154,12 @@ export type {
 export { default as Toggle } from "./components/Toggle/Toggle";
 export type { ToggleSizes } from "./components/Toggle/Toggle";
 export { default as useCarouselStyles } from "./hooks/useCarouselStyles";
-export { default as useCloseDropDown } from "./hooks/useCloseDropDown";
-export { default as useFilterBar } from "./hooks/useFilterBar";
-export { default as useMultiSelect } from "./hooks/useMultiSelect";
+// The useCloseDropDown, useFilterBar and useMultiSelect hooks will not be
+// exported until the refactor of the v2 versions of the MultiSelect and
+// FilterBar components has been completed.
+// export { default as useCloseDropDown } from "./hooks/useCloseDropDown";
```

Comment:
I think this is used in the Header and is fine to keep it exported.

Reply:
`useCloseDropDown` has been re-enabled.